tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -1,7 +1,39 @@
 import unittest
 
-from backend.corpora.common.corpora_orm import DbContributor, DbDatasetArtifact, DbDeploymentDirectory
+from backend.corpora.common.corpora_orm import (
+    DbContributor,
+    DbDatasetArtifact,
+    DbDeploymentDirectory,
+    DatasetArtifactType,
+    DatasetArtifactFileType,
+)
 from backend.corpora.common.entities.dataset import Dataset
+from tests.unit.backend.corpora.common.entities.utils import get_ids
+
+
+class DatasetParams:
+    count = 0
```

Comment:
Not sure what the purpose of this is? There isn't really a need to have the dataset names be unique.

Reply:
it will be easier to debug to know which one failed.

Reply:
We don't clear the DB after each test and some test generate multiple datasets in order to run.

Reply:
we could clear the DB after every test, but it will be significantly slower.

Reply:
Apologies, I still don't understand. What does clearing the database have to do with unique dataset names? The UUIDs are naturally going to be different because they are generated using the uuid4() function. This variable is only used for the dataset name itself and the value isn't even checked anywhere. You check the dataset.id in places but this isn't related to that.

Reply:
I use it when I'm using the python debugger to differentiate the datasets.